.github/workflows/test.yml

```diff
@@ -0,0 +1,21 @@
+name: Test
+on:
+  pull_request:
+    branches: 
+      - master
+      - develop
+      - 'release/**'
```

Comment:
Why limit testing to only these branches?

Reply:
This was a result of a copy and paste. Will fix.

Reply:
Fixed.